erpnext/selling/doctype/customer/customer_list.js

```diff
@@ -1,3 +1,10 @@
 frappe.listview_settings['Customer'] = {
 	add_fields: ["customer_name", "territory", "customer_group", "customer_type", "image"],
+
+	onload: function(listview) {
+		frappe.route_options = {
+			"customer_group": "",
```

Comment:
From where those default values are coming?

Reply:
it's set in boot.py https://github.com/frappe/erpnext/blob/develop/erpnext/startup/boot.py#L19

Reply:
from here it's set in `frappe.sys_defaults` so it is inserted in filter with global defaults.

Reply:
I have removed default territory from Selling Settings, cleared cache and reloaded the system. Still default territory is set in Customer list view in my local.